EDDiscovery/EDDOptions.cs

```diff
@@ -113,110 +107,43 @@ private void ProcessConfigVariables()
         {
             var appsettings = System.Configuration.ConfigurationManager.AppSettings;
 
-            if (appsettings["StoreDataInProgramDirectory"] == "true") StoreDataInProgramDirectory = true;
+            if (appsettings["StoreDataInProgramDirectory"] == "true")
+                StoreDataInProgramDirectory = true;
+
             UserDatabasePath = appsettings["UserDatabasePath"];
         }
 
-        private void ProcessOptionsFile()
+        private void ProcessOptionsFileOption()     // command line -optionsfile
         {
             string optionsFileName = "options.txt";
-            bool useAppDataOptionsFile = false;
 
-            ProcessCommandLineOptions((optname, optval) =>
-            {
+            ProcessCommandLineOptions((optname, optval) =>              //FIRST pass thru command line options looking
```

Comment:
This removes the implicit `options.txt` that has been present for the past few versions.